x11-libs/wxGTK/wxGTK-3.2.1.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal
+
+WXSUBVERSION=${PV}-gtk3					# 3.2.1-gtk3
+WXVERSION=${WXSUBVERSION%.*}			# 3.2.1
+WXRELEASE=${WXVERSION%.*}-gtk3			# 3.2-gtk3
+WXRELEASE_NODOT=${WXRELEASE//./}		# 32-gtk3
+
+DESCRIPTION="GTK+ version of wxWidgets, a cross-platform C++ GUI toolkit"
```

Comment:
Maybe the DESCRIPTION could be tuned a bit, but if nothing else, just GTK, not GTK+ anymore. GTK upstream dropped the `+` in the gtk4 timeline, but it's a project name so retroactive when referring it to this way.